gensim/models/atmodel.py

```diff
@@ -379,6 +381,26 @@ def inference(self, chunk, author2doc, doc2author, rhot, collect_sstats=False, c
         Avoids computing the `phi` variational parameter directly using the
         optimization presented in **Lee, Seung: Algorithms for non-negative matrix factorization, NIPS 2001**.
 
+        Parameters
+        ----------
+        chunk: int
+            The chunk numer of the sparse document vector on which inference needs to be done.
+        author2doc: dict
```

Comment:
`dict of ???`

Reply:
Is it a dict of str and str?

Reply:
`dict of (type, type)`, check by tests, what's types should be here